pkg/activator/net/throttler.go

```diff
@@ -524,3 +526,23 @@ func (ib *InfiniteBreaker) Maybe(ctx context.Context, thunk func()) error {
 		return ctx.Err()
 	}
 }
+
+// SetCapacity is a test helper that sets Throttler in a desired
```

Comment:
If it's a test helper, can we implement it in the `_test` file? Seems like only the handler test needs it anyway.

Reply:
Yeah, I left a comment on the derivative change.  This feels weird, and at a minimum should more clearly (e.g. the name?) be designated as for testing.  If it's not in the real code than even better.

Reply:
Yeah, I don't like it either.
I'll go with rename, but I can't move it to _test.go file, it won't be visible

Reply:
Done.